Automattic-Tracks-iOS/UI/Crash Logging/CrashLoggingView.swift

```diff
@@ -0,0 +1,97 @@
+import SwiftUI
+import Sentry
+
+@available(iOS 13.0, OSX 10.15, *)
+public struct CrashLoggingView: View {
+
+    @State
+    var sendErrorAndWaitStatus: SendErrorAndWaitStatus = .none
```

Comment:
Genuinely curious: I've never seen property wrappers being defined on the line above their property. Where does this style come from?

Reply:
Personally I tend to use that style a lot. I think that's even part of most [style guides](https://google.github.io/swift/#attributes) nowadays.

My 2¢ is that this helps in multiple situations:
 - It clearly separates the PW from the rest, making the declaration easier to read, especially in case you end up with multiple annotations in the future
 - It makes for simpler and nicer git diffs if/when you add/remove more PWs on the method
 - It allows to add parameters to the PW annotations without ending up with super long lines because of the longer attributes and parameters
 - It also has the benefit of being consistent with the style used for annotations in Android

Like imagine having all the attributes on one line in this example below…
```
@objc(objCFoo)
@available(*, unavailable, renamed: "bar")
@discardableResult
func foo(someParameter: Int, anotherParameter: String) {
  …
}
```

Granted, for properties that have a single and short attribute annotation like just `@State var` I still see a lot of cases where people put it on the same line "because it's short enough". But I prefer the rule to be consistent (and always put them on their own line) rather that changing the style depending on a "feeling" of if it's short enough or not and not be scalable for future changes 😉

Reply:
I just started doing it because some variable names got quite long and it was a pain in the neck to read. If I was going to be consistent I fell down on the side of separate lines, but @AliSoftware's rationale is way better than my "whoops I fell into this" approach 😄